pretyping/evardefine.mli

```diff
@@ -38,8 +38,8 @@ val split_as_array : env -> evar_map -> type_constraint ->
 val valcon_of_tycon : type_constraint -> val_constraint
 val lift_tycon : int -> type_constraint -> type_constraint
 
-val define_evar_as_product : env -> evar_map -> existential -> evar_map * types
-val define_evar_as_lambda : env -> evar_map -> existential -> evar_map * types
+val define_evar_as_product : env -> evar_map -> Names.Name.t -> existential -> evar_map * types
+val define_evar_as_lambda : env -> evar_map -> Names.Name.t -> existential -> evar_map * types
```

Comment:
A cheap way to prevent having to write overlays is to make this argument optional.